packages/fiori/test/samples/Bar.sample.html

```diff
@@ -0,0 +1,135 @@
+<header>
+	<div class="control-header">Bar</div>
+	<div class="component-heading-since">
+		<span><!--since_tag_marker--></span>
+	</div>
+</header>
+
+<div class="component-package">@ui5/webcomponents</div>
+
+<div class="control-tag">&lt;ui5-bar&gt;</div>
+
+<section>
+	<h3>Footer Bar with Content</h3>
+	<div class="snippet">
+		<ui5-bar design="Footer">
+			<div slot="startContent">
+				<ui5-button>Left Button</ui5-button>
+				<ui5-label id="basic-label">Basic Label</ui5-label>
+			</div>
+			<div slot="middleContent">
+				<ui5-button>Middle Button</ui5-button>
+				<ui5-button>Middle Button</ui5-button>
+				<ui5-button>Middle Button</ui5-button>
+				</div>
+			<div slot="endContent">
+				<ui5-button>Right Button</ui5-button>
+			</div>
+		</ui5-bar>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+<ui5-bar design="Footer">
+	<div slot="startContent">
+		<ui5-button>Left Button</ui5-button>
+		<ui5-label id="basic-label">Basic Label</ui5-label>
+	</div>
+	<div slot="middleContent">
+		<ui5-button>Middle Button</ui5-button>
+		<ui5-button>Middle Button</ui5-button>
+		<ui5-button>Middle Button</ui5-button>
+		</div>
+	<div slot="endContent">
+		<ui5-button>Right Button</ui5-button>
+	</div>
+</ui5-bar>
+	</xmp></pre>
+</section>
+<section>
+	<h3>SubHeader Bar with Content and without slot wrappers</h3>
+	<div class="snippet">
+		<ui5-bar design="SubHeader">
+			<ui5-button slot="startContent">Left Button</ui5-button>
+			<ui5-label slot="startContent" id="basic-label">Basic Label</ui5-label>
+			<ui5-button slot="middleContent">Middle Button</ui5-button>
+			<ui5-button slot="middleContent">Middle Button</ui5-button>
+			<ui5-button slot="middleContent">Middle Button</ui5-button>
+			<ui5-button slot="endContent">Right Button</ui5-button>
+		</ui5-bar>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+<ui5-bar design="SubHeader">
+	<ui5-button slot="startContent">Left Button</ui5-button>
+	<ui5-label slot="startContent" id="basic-label">Basic Label</ui5-label>
+	<ui5-button slot="middleContent">Middle Button</ui5-button>
+	<ui5-button slot="middleContent">Middle Button</ui5-button>
+	<ui5-button slot="middleContent">Middle Button</ui5-button>
+	<ui5-button slot="endContent">Right Button</ui5-button>
+</ui5-bar>
+	</xmp></pre>
+</section>
+<section>
+	<h3>FloatingFooter Bar with Content</h3>
+	<div class="snippet">
+		<ui5-bar design="FloatingFooter">
```

Comment:
Remove the nesting from the sample page (in the test page it is fine to have them) to advertise the direct usage.